### PR TITLE
fixing test_failure_threshold_deletions

### PR DESCRIPTION
**Pull request by bdeggleston**
This fixes one of the failing tests in CASSANDRA-10267. The log pattern we grep for has changed in 3.0, and the read_request_timeout was reduced to prevent sporadic OperationTimedOut exceptions, instead of ReadTimeout

### CODE REVIEW

File: upgrade_tests/paging_test.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
+import itertools
 import time
 import uuid
 
-from cassandra import ConsistencyLevel as CL
+from cassandra import ConsistencyLevel as CL, OperationTimedOut
```
Comment (aboudreault) [inline upgrade_tests/paging_test.py]: Is this used?

File: upgrade_tests/paging_test.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 from cassandra import InvalidRequest, ReadTimeout
 from cassandra.query import SimpleStatement, dict_factory, named_tuple_factory
 from dtest import run_scenarios, debug
-from tools import since, require, rows_to_list
+from tools import since, require, rows_to_list, no_vnodes
```
Comment (aboudreault) [inline upgrade_tests/paging_test.py]: Is this used?

**Comment by aboudreault** [comment]
LGTM, just 2 minor comments and the lint to fix.

**Comment by bdeggleston** [comment]
thanks, I just pushed up a fix

**Pull request merged**